### PR TITLE
run rustfmt on librustc_incremental folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

**Comment by srinivasreddy** [comment]
@arielb1 LGTM. What do you think?

File: src/librustc_incremental/assert_dep_graph.rs [hunk 1]
```diff
@@ -272,10 +267,14 @@ impl<'a, 'tcx, 'q> dot::Labeller<'a> for GraphvizDepGraph<'q> {
...
-                              .map(|c| if c == '_' || c.is_alphanumeric() { c } else { '_' })
-                              .collect();
+        let s: String = format!("{:?}", n)
+            .chars()
+            .map(|c| if c == '_' || c.is_alphanumeric() {
+                c
+            } else {
+                '_'
+            })
+            .collect();
```
Comment (eddyb) [inline src/librustc_incremental/assert_dep_graph.rs]: @nrc @ubsan This looks harder to read.

Comment (nrc) [inline src/librustc_incremental/assert_dep_graph.rs]: Yeah, this is probably the trickiest corner of formatting at the moment - I prefer visual indent for method chains, but it makes closures either ugly or subject to extreme rightward drift.

**Comment by nrc** [comment]
r = me with a rebase

**Comment by frewsxcv** [comment]
Closing due to inactivity. @srinivasreddy If you're still interested in these changes, let me know and I can reopen, otherwise you could just open a new PR.

**Pull request closed without merging**